<?xml version="1.0"?>
<robot name="steel_table">
  <link name="base_link">
  <contact>
      <lateral_friction value="1"/>
  </contact>
    <inertial>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <mass value="0" />   <!-- was 415 -->
      <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0" />
    </inertial>
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="panda_table.obj"/>
      </geometry>
      <material name="grey">
        <color rgba="0.7 0.7 0.7 1.0" />
      </material>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="panda_table.obj"/>
      </geometry>
    </collision>
   </link>
</robot>
